reeds/function_libs/optimization/eds_energy_offsets.py

```diff
@@ -94,10 +94,11 @@ def estimate_energy_offsets(ene_trajs: List[pd.DataFrame], initial_offsets: List
     else: 
         undersampling_eoffs = all_eoffs[undersampling_idx:]
         results = 'Undersampling found at replica ' + str(undersampling_idx+1) \
-                  + ' with s = ' + str(s_values[undersampling_idx]) + '\n\n' 
```

Comment:
is this correct? It's a bit counter-intuitive, that the undersampling index is `undersampling_idx+1` but the location in the array is `undersampling_idx-1`